_posts/2020-11-04-differences-between-programming-languages.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: post
+title:  "Differences between Programming Languages"
+date:   2020-11-04 12:50:28
+categories: jekyll update
+---
+This page introduces all programming languages and compares the different programming languages in terms of the coding style, language principles, related applications, advantages and disadvantages, as well as how the language is executed. The purpose of making this page is to let you be familiar with all these programming languages and help you choose the one you mostly want to learn and use.
+
+# List of Common Programming Languages
+- Python
+- Java/JavaScript
+- Ruby
+- Bash
+- HTML
+- C/C++
+- SQL
+
+# Introduction of Common Programming Languages
+## 1. Python
+![python](https://beat0154.github.io/Python-Programming-Portfolio/python.png)
+
+Python is a high-level programming language. It supports a variety of programming paradigms such as functional programming, object-oriented programming and procedural programming. Python is highly extensible, which serves as a feature of adding programmable interfaces to many applications.
+
+Python is an easily understandable and readable language. If you are new to the programming language, Python would be a nice choice. Unlike other languages, it uses whitespace indentation to delimit blocks.[^1]
+
+### Python History
+Python was designed by *Guido van Rossum* in the late 1980s. Python *2.0* was released on October 2000 with the new feature of containing a cycle-detecting garbage collector. Python *3.0* was released on December 2000. New version will be released on May 2022.[^1]
+![Python](https://www.cwi.nl/news/blogs/interview-guido-van-rossum-201cid-rather-write-code-than-papers.201d/@@images/58e10972-b372-4cec-9e1f-a265cf227e78.jpeg)
+
+### Python programming examples
+```
+counter = 1
+while counter < 5:
+  print("The current counter value: " % counter)
+  counter = counter + 1
+```
```

Comment:
@DingShen29 I'm sorry for approving the changes before further thought. I spent more time thinking about your submission. Do you think it would be helpful to explain what the example code is doing? A beginner coder might not be able to understand what the example code is for.